client/containers/AddTravelerContainer.jsx

```diff
@@ -14,24 +14,33 @@ export class AddTravelerContainer extends React.Component {
   handleSubmit(e) {
     e.preventDefault();
 
-    const formValues = this.props.form.adminAddTraveler.values;
+    const formValues = this.props.form.travelerForm.values;
 
-    if(flightInfoSubmitted(formValues)) {
+    if (flightInfoSubmitted(formValues)) {
       const { flightNum, airlineCode, arrivalTime } = formValues;
       const day = arrivalTime.getDate();
       const year = arrivalTime.getYear() + 1900;
       const month = arrivalTime.getMonth() + 1;
-      this.props.checkFlight(airlineCode, flightNum, year, month, day);
+      this.props.checkFlight(airlineCode, flightNum, year, month, day)
+      .then((res) => {
+        formValues.countryCode = formValues.countryCode.split('-')[1].slice(2);
```

Comment:
This could be DRY-ed up (similar chaining on line 33); also if it's encapsulated in a function, it'll be easier to follow what's going on (I'm not 100% sure just from looking at this)